warp/src/crypto/mod.rs

```diff
@@ -160,6 +157,8 @@ impl From<ed25519_dalek::SecretKey> for DID {
 
 pub fn generate<const N: usize>() -> [u8; N] {
     let mut buf = [0u8; N];
-    getrandom::getrandom(&mut buf).unwrap();
+    for x in buf.iter_mut() {
+        *x = rand::random()
+    }
```

Comment:
You could use `RngCore::fill_bytes` instead (would require importing `OsRng` and `RngCore`)

```suggestion
    OsRng.fill_bytes(&mut buf);
```

Reply:
`OsRng` uses `getrandom` so it'll beat the point of this change.

Reply:
rand uses getrandom by default anyway the last I check (though getrandom may be more reliable to use for seeding), otherwise you need to disable the default features and use `std` and `std_rng` features and maybe change it to 

```rust
    let mut rng = rand::thread_rng();
    rng.fill_bytes(&mut buf);
```

Reply:
> rand uses getrandom by default anyway the last I check

I did look at the code before my previous comment, and it doesn't. See [OsRng docs](https://docs.rs/rand/latest/rand/rngs/struct.OsRng.html).

Reply:
Looking at rand itself, `std` feature uses `getrandom`

See https://github.com/rust-random/rand/blob/0.8.5/Cargo.toml#L36

Reply:
Ah, you're right. Thanks.